pkg/installmanager/installmanager.go

```diff
@@ -950,14 +973,99 @@ func (m *InstallManager) generateAssets(cd *hivev1.ClusterDeployment, workerMach
 	return nil
 }
 
+// TODO: Combine this with patchWorkerMachineSetManifest once HIVE-2397 is completed
```

Comment:
...and then you could inline this func.

Also, I think this comment was intended to go with the next func.